aureliafile.js

```diff
@@ -0,0 +1,50 @@
+var cli = require('aurelia-cli');
+
+var config = {
+  js: {
```

Comment:
Let's add the default here as a way of documenting the options, yes. Out to the side, whenever it is a default, just add a little comment to that effect.